packages/edit-site/src/components/global-styles/screen-revisions/style.scss

```diff
@@ -8,62 +8,80 @@
 	margin: 0;
 	li {
 		margin-bottom: 0;
-		border-left: 1px solid $gray-300;
 	}
 }
 
 .edit-site-global-styles-screen-revisions__revision-item {
 	position: relative;
-	padding: $grid-unit-10 0 $grid-unit-10 $grid-unit-15;
+	padding-left: $grid-unit-20;
+	overflow: hidden;
+	cursor: pointer;
 
-	&:first-child {
-		padding-top: 0;
+	&:hover {
+		background: rgba(var(--wp-admin-theme-color--rgb), 0.04);
+		.edit-site-global-styles-screen-revisions__date {
+			color: var(--wp-admin-theme-color);
+		}
 	}
 
-	&:last-child {
-		padding-bottom: 0;
+	&::before,
+	&::after {
+		position: absolute;
+		content: "\a";
+		display: block;
 	}
 
 	&::before {
 		background: $gray-300;
 		border-radius: 50%;
-		content: "\a";
-		display: inline-block;
 		height: $grid-unit-10;
 		width: $grid-unit-10;
-		position: absolute;
-		top: 50%;
-		left: 0;
+		top: $grid-unit-20 + 2;
+		left: $grid-unit-20 + 1; // So the circle is centered on the line.
 		transform: translate(-50%, -50%);
+		z-index: 1;
 	}
 	&.is-selected::before {
 		background: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
 	}
-}
 
-.edit-site-global-styles-screen-revisions__revision-button {
-	width: 100%;
-	height: auto;
-	display: block;
-	padding: $grid-unit-10 $grid-unit-15;
+	&::after {
+		height: 100%;
+		left: $grid-unit-20;
+		top: 0;
+		width: 0;
+		border: 0.5px solid $gray-300;
+	}
 
-	&:hover {
-		background: rgba(var(--wp-admin-theme-color--rgb), 0.04);
+	&:first-child::after {
+		top: $grid-unit-20 + 2;
+	}
 
-		.edit-site-global-styles-screen-revisions__date {
-			color: var(--wp-admin-theme-color);
+	&:last-child::after {
+		height: $grid-unit-20 + 2;
+	}
+
+	// Nested to override specificity of .components-button.
+	.edit-site-global-styles-screen-revisions__revision-button {
+		width: 100%;
+		height: auto;
+		display: block;
+		padding: $grid-unit-15 $grid-unit-15 $grid-unit-15 $grid-unit-30;
+		&:focus,
+		&:active {
+			outline: 0;
+			box-shadow: none;
 		}
 	}
 }
 
 .is-selected {
+	color: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
+	background: rgba(var(--wp-admin-theme-color--rgb), 0.04);
```

Comment:
These color and background styles for the `.is-selected` modifier is not scoped to a specific `.edit-site-global-styles-screen-revisions__` class, so it's affecting every component that uses a `is-selected` modifier. For example, the `PaletteEdit` in the Global Styles color palette editor:

<img width="274" alt="Affected PaletteEdit" src="https://github.com/WordPress/gutenberg/assets/555336/744d24b0-0fd2-491b-8a8b-0b203100615d">

@ramonjd I don't have much context on these Screen Revisions UIs — do you have bandwidth to scope this CSS properly to the relevant classes?

Reply:
Oh, great catch! I can look into this.

Reply:
Fix here: https://github.com/WordPress/gutenberg/pull/58228

Reply:
🤦🏻  Thanks for catching @mirka and for the quick fix @andrewserong